src/main/java/org/example/lesson10/stringManipulator/StringManipulator.java

```diff
@@ -0,0 +1,18 @@
+package org.example.lesson10.stringManipulator;
+
+public class StringManipulator {
+
+    public String removeCharacters(String str, char[] charsToRemove) {
```

Comment:
In general solution is correct but quite difficult to read. If you think how to do it without i and k counters it would be like this:
```
public String removeCharacters(String str, char[] charsToRemove) {
    StringBuilder strAfterModifying = new StringBuilder();
    for (char ch : str.toCharArray()) {
        boolean found = false;
        for (char charToRemove : charsToRemove) {
            if (Character.toLowerCase(ch) == Character.toLowerCase(charToRemove)) {
                found = true;
                break;
            }
        }
        if (!found) {
            strAfterModifying.append(ch);
        }
    }
    return strAfterModifying.toString();
}
```